client/a8c-for-agencies/components/layout/index.tsx

```diff
@@ -12,20 +12,23 @@ type Props = {
 	title: ReactNode;
 	wide?: boolean;
 	withBorder?: boolean;
+	compact?: boolean;
 };
 
 export default function Layout( {
 	children,
 	className,
 	title,
-	wide = false,
-	withBorder = false,
+	wide,
+	withBorder,
+	compact,
 	sidebarNavigation,
 }: Props ) {
 	return (
 		<Main
 			className={ classNames( 'a4a-layout', className, {
 				'is-with-border': withBorder,
+				'is-compact': compact,
```

Comment:
The spacing for the **Checkout** and **Pressable/Wordpress.com** hosting page has different spacing. By using a flag, we can determine if we need a spacing of 48px or 28px.